### PR TITLE
Fix language usage

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @LukasReschke @rullzer

### CODE REVIEW

**Comment by MorrisJobke** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
Mind fixing this one as well while at it .....

```
.........PHP Fatal error:  Call to undefined method OC_Util::generateRandomBytes() in /home/travis/build/owncloud/core/apps/contacts/tests/unit/lib/backend/mock.php on line 165

PHP Stack trace:

PHP   1. {main}() /home/travis/.phpenv/versions/5.4.37/bin/phpunit:0

PHP   2. PHPUnit_TextUI_Command::main() /home/travis/.phpenv/versions/5.4.37/bin/phpunit:722

PHP   3. PHPUnit_TextUI_Command->run() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/TextUI/Command.php:104

PHP   4. PHPUnit_TextUI_TestRunner->doRun() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/TextUI/Command.php:152

PHP   5. PHPUnit_Framework_TestSuite->run() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/TextUI/TestRunner.php:398

PHP   6. PHPUnit_Framework_TestSuite->run() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestSuite.php:716

PHP   7. PHPUnit_Framework_TestCase->run() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestSuite.php:716

PHP   8. PHPUnit_Framework_TestResult->run() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestCase.php:693

PHP   9. PHPUnit_Framework_TestCase->runBare() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestResult.php:609

PHP  10. PHPUnit_Framework_TestCase->runTest() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestCase.php:737

PHP  11. ReflectionMethod->invokeArgs() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestCase.php:860

PHP  12. OCA\Contacts\AddressBookTest->testAddChild() phar:///home/travis/.phpenv/versions/5.4.37/bin/phpunit/phpunit/Framework/TestCase.php:860

PHP  13. OCA\Contacts\Addressbook->addChild() /home/travis/build/owncloud/core/apps/contacts/tests/unit/lib/addressbookTest.php:84

PHP  14. OCA\Contacts\Contact->save() /home/travis/build/owncloud/core/apps/contacts/lib/addressbook.php:275

PHP  15. OCA\Contacts\Backend\Mock->createContact() /home/travis/build/owncloud/core/apps/contacts/lib/contact.php:306

```

**Comment by nickvergessen** [comment]
Done, but I can't seem to fix the local run, when I add the group DB, it fails with undefined methods and so on.

**Comment by MorrisJobke** [comment]
Postgres sorting failure... -.-

File: tests/unit/lib/backend/mock.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ function createAddressBook(array $properties, array $options = array()) {
 			return false;
 		}
 
-		$id = \OC_Util::generateRandomBytes('4');
+		$id = \OC::$server->getSecureRandom()->generate('4');
```
Comment (DeepDiver1975) [inline tests/unit/lib/backend/mock.php]: this call does not work on 8.2

Comment (nickvergessen) [inline tests/unit/lib/backend/mock.php]: Well, then we reached the point in time, where we can not use the same version of the app for all server versions?

Comment (DeepDiver1975) [inline tests/unit/lib/backend/mock.php]: For all - no. But all not-eol.

Comment (rullzer) [inline tests/unit/lib/backend/mock.php]: fixed

**Comment by rullzer** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
THX @rullzer

**Pull request merged**